tests/suites/manual/task.sh

```diff
@@ -4,6 +4,11 @@ test_manual() {
 		return
 	fi
 
+	if [[ ${BOOTSTRAP_PROVIDER:-} == "aws" ]]; then
```

Comment:
it feels like something else is broken if this is required. though a merge pr is not the place to fix it.

Reply:
This is only required for pipelines that make use of the aws cli (which is fairly rare) and resolves the edge case where `aws` is configured to use a different region then the integration tests.

Perhaps it should be in main.sh, ut as you say, snot something to fix in a merge PR